### ISSUE TITLE
Balancing SVD precision rifle

### ISSUE THREAD

**Issue by Xenonauts** [Xenonauts/FullSovietWeaponsExtensionPack#13]
I think this one should have more damage and mitigation than G3 SG/1, larger range but all its firing modes should cost signifcantly more, maybe it should also have worse reaction time modifier. With it you need to find perfect shooting position; when you move you "cannot" fire and should wait until next turn -- inspired by the way how a sniper rifle usually worked in XCOM 2012 by Firaxis.

**Comment by DanrVazl** [comment]
So pretty much almost 100% TU cost (a bit lower, to let the sniper actually turn from side to side) to fire the weapon? Can do!

**Comment by Xenonauts** [comment]
Yes, of course. Maybe two different firing mods (however this would break the idea of 'positional' sniper rifle)

**Comment by DanrVazl** [comment]
I'll do some experimenting and see what I come up with :)

**Comment by erutan** [comment]
I realize you're probably not updating this anymore, but I decided to give CE a shot since I'm waiting on X2 & PP to mature. :)

I'm playing with it at 92% of TUs to allow a bit more movement side to side, at the beginning of the game you can be left with 2-3 TUs at 95% outsidefrom firing which can be a bit restrictive.
